pum.py

```diff
@@ -0,0 +1,13 @@
+def pum(lcm, cmv):
+    pum = [[False for _ in range(len(lcm[0]))] for _ in range(len(lcm))]
+    for i in range(len(lcm)):
+        for j in range(len(lcm[i])):
+            operator = lcm[i][j]
+            match operator:
+                case "ANDD":
+                    pum[i][j] = cmv[i] and cmv[j]
+                case "ORR":
+                    pum[i][j] = cmv[i] or cmv[j]
+                case "NOTUSED":
```

Comment:
The lab description says that if LCM is NOTUSED then PUM should be set to true, not false. Same thing applies to `test_pum_notused()`. Apart from that, it looks good!